M3P8 Algebra III/M3P8.tex

```diff
@@ -2194,7 +2194,7 @@ \subsection{The main result}
 \begin{proof}
 Let $ r $ be a nonzero element of $ R $. Have map
 $$ \function{K\sbr{X}}{R}{X}{r}. $$
-Let $ I $ be the kernel. Since $ R $ is finite-dimensional $ I = 0 $. Since $ R $ is an integral domain $ I $ is prime. So
+Let $ I $ be the kernel. Since $ R $ is finite-dimensional, $ I \neq 0 $. Since $ R $ is an integral domain $ I $ is prime. So
```

Comment:
For consistency, let's use `\ne` rather than `\neq`, and let's put a comma in `... an integral domain $ I $ is prime. ...`